src/raft_server.c

```diff
@@ -758,7 +791,11 @@ int raft_recv_requestvote(raft_server_t* me_,
         {
             e = raft_vote_for_nodeid(me_, vr->candidate_id);
             if (0 != e)
+            {
+                log_info(me_, node, "rejected requestvote%s for %d: could not update vote: %d",
```

Comment:
would this be more appropriate as a log_error()? Similar to the case when raft_set_current_term() fails?

Reply:
Yes.